### ISSUE TITLE
minPadding and rests

### ISSUE THREAD

**Issue by foxbunny** [paulrosen/abcjs#684]
When using the `initialClef` option, the initial element on the staff gets rendered right at the edge of the staff. I've tried using `minPadding` to address this, but it does not affect rests. Here's the screenshot:

![image](https://user-images.githubusercontent.com/65413/119831827-f026a580-befd-11eb-87bc-02cdc070f20c.png)

Is this by design? And if so, what's the rationale?

Since I've provided the patch for initial clef, I could also look into adjusting the padding of the initial elements regardless of `minPadding` in cases where `initialClef` is used, which I think would be the right way to fix this.

**Comment by paulrosen** [comment]
If there were a clef there it would be in the right place. I guess that instead of just skipping the clef the x-coordinate needs to be bumped up in that case.

**Comment by foxbunny** [comment]
Yeah, that's a good point. I'll take a look at this again at some point.
